lib/imapoptions

```diff
@@ -83,6 +83,13 @@ Blank lines and lines beginning with ``#'' are ignored.
 For boolean and enumerated options, the values ``yes'', ``on'', ``t'',
 ``true'' and ``1'' turn the option on, the values ``no'', ``off'',
 ``f'', ``false'' and ``0'' turn the option off.
+.PP
+Duration options take the form of a number followed by a unit, for example
+\fB32m\fR (32 minutes).  Units are \fBd\fR (days), \fBh\fR (hours), \fBm\fR
+(minutes) and \fBs\fR (seconds).  Multiple units can be combined and will
```

Comment:
Nit: Days not necessarily have a duration of 24 hours, but are hard-coded as such in the current parser implementations. It would be unreasonable to support day light saving time for evaluation of the configured items, but let's just be explicit here and state "Day durations are multiples of 24 hours"?